codegen/cmd/injection-gen/args/args.go

```diff
@@ -49,6 +49,8 @@ func (ca *CustomArgs) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&ca.ListerHasPointerElem, "lister-has-pointer-elem", false, "")
 	fs.MarkDeprecated("lister-has-pointer-elem", "this flag has no effect")
+
+	fs.Bool("skipInitFuncForInformer", false, "Skip the init function for informer")
```

Comment:
We should wire this to the `CustomArgs` structs like we do for the other flags

Also can we follow the flag naming convention we are using  which is `snake-case`

I'd probably call this `disable-informer-init`

Reply:
@dprotaso ok done